tests/io-test/slightfile.toml

```diff
@@ -0,0 +1 @@
+specversion = "0.2"
```

Comment:
unused file?

Reply:
Not really. I've kept it around to provide a working set of files local to the io-test slight app. I'd like to keep it around until #355 is resolved.